tests/publiccloud/path_and_reboot.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Refresh repositories, apply patches and reboot
+#
+# Maintainer: Pavel Dostal <pdostal@suse.cz>
+
+use base 'consoletest';
+use registration;
+use warnings;
+use testapi;
+use strict;
+use utils;
+
+sub run {
+    my ($self, $args) = @_;
+    select_console 'root-console';
+
+    $args->{my_instance}->run_ssh_command(cmd => "sudo zypper ref");
+    ssh_fully_patch_system($args->{my_instance}->public_ip);
+    $args->{my_instance}->softreboot();
```

Comment:
Do we need to setup 'root-console' and ssh port forwarding afterwards?

Reply:
We do that. That is why I extracted that part from `ssh_interactive_init` into `ssh_interactive_start` module.

Reply:
ok